Bundle/BlogBundle/Resources/views/Blog/Tabs/_articles.html.twig

```diff
@@ -38,7 +38,7 @@
                 </a>
 
                 {% if article.author is not null %}
-                    <em>{{ article.author.firstname }} {{ article.author.lastname }}</em>
+                    <em>{{ article.author.firstname is defined ? article.author.firstname : "" }} {{ article.author.lastname is defined ? article.author.lastname : "" }}</em>
```

Comment:
@MadeWilson I prefer the `article.author.firstname|default('')` personally but this syntax is right too.